src/send_kcidb.py

```diff
@@ -570,6 +573,14 @@ def _run(self, context):
                 }
             }
             self._send_revision(context['client'], revision)
+
+            # mark the node as processed, sent_kcidb field to True
+            # TBD: job nodes might have child nodes, mark them as processed as well
+            node['sent_kcidb'] = True
```

Comment:
I believe the field name is `kcidb_sent` from PR https://github.com/kernelci/kernelci-core/pull/2746/.

Reply:
Sure, np. I'd be happy to help in case you find something unclear in the bridge service.